### DISCUSSION TITLE
Disable bounce when reaching web page end on iOS ipadOS devices

### DISCUSSION THREAD

**Discussion by zpaolo11x** [gotson/komga#1600] [Ideas]
### Describe your suggested feature

On iPhone and iPad when the web page is at the top or bottom, further vertical swipe will introduce a "bounce" spring-loaded effect, that if forced causes the page to reload. Especially when using the web reader this can be a bit distracting. There are ways to disable this using CSS like this (which disable it on the whole website):

html {
    overscroll-behavior: none;
}

or more complex solutions like this: https://github.com/timbartsch/no-bounce 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
